ProcessHoloFiles.ps1

```diff
@@ -77,52 +38,7 @@ function Select-Folder([string]$description) {
     }
 }
 
-
-# Prompt the user to select the folder containing .holo files
-$holoFolderPath = Select-Folder -description "Select the folder containing .holo files"
-# Check if the user selected a folder
-if (-not $holoFolderPath) {
-    Write-Host "No folder was selected. Exiting script." -ForegroundColor Red
-    exit
-}
-
-# Get the list of configuration files from the user (only if the first one was selected)
-$configFiles = Get-ConfigFiles
-
-$exePath1 = "Holovibes.exe"
-$exePath2 = "build/bin/Holovibes.exe"
-$exePath = ""
-
-# Check if Holovibes.exe exists
-if (Test-Path $exePath1) {
-    $exePath = $exePath1
-} else {
-    $exePath = $exePath2
```

Comment:
Maybe keep that

Reply:
It's my bad i only tested with release, i just added the bin/build exe path

Reply:
Thanks!